### PR TITLE
Report-device-specific-scanning-volume-into-web [BIPI-5976]

### PR DESCRIPTION
**Pull request by ZuzanaMikulova**
ROS service for sending device scanning volume has been created.
Web part needs to be added, then it can be tested.

### CODE REVIEW

**Review by mikiPhotoneo — APPROVED** [review]
It looks good :)

However it has one limitation (caused by PhoXi API) - it returns scannig volume only after scan was triggered. We need to come up with some workflow..

Please do not merge yet

File: src/RosInterface.cpp [hunk 1]
```diff
@@ -236,6 +237,35 @@ namespace phoxi_camera {
...
+                }
+            }
+            pcl_conversions::moveFromPCL(meshPcl, res.mesh);
+            res.success = true;
+            res.message = OKRESPONSE;
+        } catch (PhoXiInterfaceException& e) {
+            res.success = false;
+            res.message = e.what();
+        }
+        return true;
```
Comment (AdamSojkaPho) [inline src/RosInterface.cpp]: I would appreciate if you could specify what kind of conversion is happening here. Why we can't transfer the pho::api::PhoXiMesh directly to pcl_msgs::PolygonMesh?

**Review by AdamSojkaPho — CHANGES_REQUESTED** [review]
Document the algorigthm. Encapsulate the mesh conversion.

File: src/RosInterface.cpp [hunk 2]
```diff
@@ -24,6 +24,7 @@ namespace phoxi_camera {
         triggerImageService =nh.advertiseService("trigger_image", &RosInterface::triggerImage, this);
         getFrameService = nh.advertiseService("get_frame", &RosInterface::getFrame, this);
         saveFrameService = nh.advertiseService("save_frame", &RosInterface::saveFrame, this);
+        getScanVolumeService = nh.advertiseService("get_scan_volume", &RosInterface::getScanningVolume, this);
```
Comment (mikiPhotoneo) [inline src/RosInterface.cpp]: please rename `get_scan_volume` service to `get_scanning_volume`, thanks

Comment (ZuzanaMikulova) [reply src/RosInterface.cpp]: Fixed now

File: src/RosInterface.cpp (hunk 1 above)
Comment (ZuzanaMikulova) [reply src/RosInterface.cpp]: Encapsulated and slightly changed for better readability.

**Review by mikiPhotoneo — APPROVED** [review]

**Review by AdamSojkaPho — APPROVED** [review]
The new function now helps to explain what the code is for. Thanks. 
LGTM

**Pull request merged**